src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class Rectangle implements Figure {
+    private String color;
+    private double height;
+    private double width;
+
+    public Rectangle(String color, double height, double width) {
+        this.color = color;
+        this.height = height;
+        this.width = width;
+    }
+
+    @Override
+    public double getArea() {
+        return height * width;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Rectangle: color="
+                + color
+                + ", height="
+                + height
+                + ", width="
+                + width
+                + ", area="
+                + getArea());
+    }
```

Comment:
Make sure to avoid using magic numbers or strings in your code. If you have any, replace them with named constants for better readability and maintainability.